carma/rviz/carma_default_ros2.rviz

```diff
@@ -9,23 +9,16 @@ Panels:
         - /LaneLet2 Map1/Topic1
```

Comment:
Can we add a bullet point list in the PR description to list out all of the updates made to the rviz config?